### PR TITLE
Fix acceptance tests

### PR DESCRIPTION
**Pull request by lgonczar**

### CODE REVIEW

File: data/non-private-individual-owner/AGL1000_non_private_individual.json [hunk 1]
```diff
@@ -582,7 +582,6 @@
 
   },
   "filed_plan_format": "UNKNOWN",
-  "edition_date": "1996-07-01",
```
Comment (lgonczar) [inline data/non-private-individual-owner/AGL1000_non_private_individual.json:585]: Duplicate element

File: features/step_definitions/official_copy_legal_txt.rb [hunk 2]
```diff
@@ -4,15 +4,13 @@
...
 
-Then(/^I can see the edition dates in the correct format$/) do
-  expect(@pdf_pages[0]).to include('20 December 2015')
+Then(/^I can see the edition date in the correct format$/) do
+  expect(@pdf_pages[0]).to match('20\s?December\s?2015')
 end
 
 Then(/^I can see the dates of the entries on the register in the correct format$/) do
-	expect(@pdf_pages[0]).to include('6 November 1995')
-	expect(@pdf_pages[0]).to include('28 August 2014')
```
Comment (lgonczar) [inline features/step_definitions/official_copy_legal_txt.rb:13]: Two of these dates don't exist in the source JSON file

**Comment by russwillis** [comment]
Looks good so far. Just running them locally to see what happens. 2 failed but should have passed. Heather has retried the same tests and they passed. Happy to merge.

**Comment by Pheonnexx** [comment]
LGTM

**Pull request merged**